apps/src/aichat/types/chatWorkspace.ts

```diff
@@ -0,0 +1,4 @@
+export enum WorkspaceTeacherViewTab {
```

Comment:
@kelbyhawn and I thought about where this `enum` belongs in existing categories and it didn't seem to belong in any of them. So similar to `chatButton`, create a new file for this. 😃